multicluster/apis/multicluster/v1alpha1/clusterset_types.go

```diff
@@ -27,14 +27,23 @@ type MemberCluster struct {
 	ClusterID string `json:"clusterID,omitempty"`
 	// API server of the destination cluster.
 	Server string `json:"server,omitempty"`
-	// Secret name to access API server of the member from the leader cluster.
+	// Secret name in the member cluster which will be used to access API
+	// server of the leader cluster.
 	Secret string `json:"secret,omitempty"`
-	// ServiceAccount used by the member cluster to access into leader cluster.
+	// ServiceAccount indicates which account is bound by the member cluster's access token
+	// to access the leader cluster. This is an optional field which helps admin to check
+	// which account is used by corresponding member cluster to access the leader cluster.
 	ServiceAccount string `json:"serviceAccount,omitempty"`
```

Comment:
If it does not have any real use, should we deprecate this field (and remove it from documentation)?

Reply:
@jianjuns Do you mean to add a `DEPRECATED` comment on the field? or remove this field directly?

Reply:
Add a `DEPRECATED` comment